<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/помощник викторины/викторина.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>картинкни/викторина.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="math">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;</string>
    </property>
    <property name="text">
     <string>Математика</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="russ">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;</string>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logic">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;</string>
    </property>
    <property name="text">
     <string>логика</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="history">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;</string>
    </property>
    <property name="text">
     <string>История</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;</string>
    </property>
    <property name="text">
     <string>БАЛАНС:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rage Italic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
font: 18pt &quot;Rage Italic&quot;;</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="m_100">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_100">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(1, 1, 1, 0);
background-color:  rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_100">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_100">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>133</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_200">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_200">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_200">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>133</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_200">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_300">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_300">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_300">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>133</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_300">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_400">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_400">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_400">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>133</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_400">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>137</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgba(255, 255, 255, 0);
color: rgba(1, 1, 1, 0)</string>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>math</zorder>
   <zorder>russ</zorder>
   <zorder>logic</zorder>
   <zorder>history</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_11</zorder>
   <zorder>m_100</zorder>
   <zorder>r_100</zorder>
   <zorder>log_100</zorder>
   <zorder>h_100</zorder>
   <zorder>log_200</zorder>
   <zorder>m_200</zorder>
   <zorder>h_200</zorder>
   <zorder>r_200</zorder>
   <zorder>log_300</zorder>
   <zorder>m_300</zorder>
   <zorder>h_300</zorder>
   <zorder>r_300</zorder>
   <zorder>log_400</zorder>
   <zorder>m_400</zorder>
   <zorder>h_400</zorder>
   <zorder>r_400</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="помощник викторины.qrc"/>
 </resources>
 <connections/>
</ui>
